src/typer/kvittering.ts

```diff
@@ -0,0 +1,4 @@
+export interface IKvittering {
+    tekst: string;
+    mottattDato: string;
```

Comment:
Er det mye herk å gjøre mottattDato til et datetime-type objekt istedenfor string?

Reply:
Kan teste

Reply:
Tror ikke det vil fungere, da må dere ha en mapper isåfall. Tenker string er fint.